src/libraries/System.Security.Cryptography.X509Certificates/tests/RevocationTests/AiaTests.cs

```diff
@@ -16,21 +16,26 @@ public static void EmptyAiaResponseIsIgnored()
                 PkiOptions.AllRevocation,
                 out RevocationResponder responder,
                 out CertificateAuthority root,
-                out CertificateAuthority intermediate,
+                out CertificateAuthority[] intermediates,
                 out X509Certificate2 endEntity,
+                intermediateAuthorityCount: 2,
                 pkiOptionsInSubject: false);
 
+            CertificateAuthority intermediate1 = intermediates[0];
+            CertificateAuthority intermediate2 = intermediates[1];
+
             using (responder)
             using (root)
-            using (intermediate)
+            using (intermediate1)
+            using (intermediate2)
```

Comment:
Why not use the variable declaration using for these?

Reply:
Uhhhhhhhh, because I... didn't. Will fix when I see what's up with Windows.

(I was sure I could just get this right on the first try from my Linux box..)